build/buildRunTimeImageBases.sh

```diff
@@ -52,13 +52,6 @@ docker build \
     -t "oryxdevmcr.azurecr.io/private/oryx/$RUNTIME_BASE_IMAGE_NAME-stretch" \
     $REPO_DIR
 
-docker build \
```

Comment:
Could you please clarify: Is Buster actually creating the secure supply chain issue? I thought it was the Stretch and PHP.5-6 images.
Won't we have the need to rebuild some images on that base in the future (Buster EOL is still far away)?